recipes/mupdf/Makefile-shared-libs-osx.patch

```diff
@@ -0,0 +1,39 @@
+--- Makefile.orig	2020-03-25 11:02:34.874709800 -0500
++++ Makefile	2020-04-07 09:39:20.576540717 -0500
+@@ -20,7 +20,7 @@
+ # Do not specify CFLAGS or LIBS on the make invocation line - specify
+ # XCFLAGS or XLIBS instead. Make ignores any lines in the makefile that
+ # set a variable that was set on the command line.
+-CFLAGS += $(XCFLAGS) -Iinclude
++CFLAGS += $(XCFLAGS) -Iinclude -fPIC
+ LIBS += $(XLIBS) -lm
+ 
+ ifneq ($(threading),no)
+@@ -189,17 +189,21 @@
+ 
+ # --- Library ---
+ 
+-MUPDF_LIB = $(OUT)/libmupdf.a
+-THIRD_LIB = $(OUT)/libmupdf-third.a
+-THREAD_LIB = $(OUT)/libmupdf-threads.a
+-PKCS7_LIB = $(OUT)/libmupdf-pkcs7.a
++MUPDF_LIB = $(OUT)/libmupdf.dylib
++THIRD_LIB = $(OUT)/libmupdf-third.dylib
++THREAD_LIB = $(OUT)/libmupdf-threads.dylib
++PKCS7_LIB = $(OUT)/libmupdf-pkcs7.dylib
+ 
+-$(MUPDF_LIB) : $(MUPDF_OBJ)
++$(MUPDF_LIB) : $(MUPDF_OBJ) $(THIRD_LIB) $(THREAD_LIB)
++	$(LINK_CMD) -shared -Wl,-install_name -Wl,libmupdf.dylib -undefined dynamic_lookup $(THIRD_LIBS)
```

Comment:
Does this use `LDFLAGS`?

Reply:
Again, this is part of an upstream patch

Reply:
Sure, but is `LDFLAGS` used at all?

Reply:
Yes, in the Makefile, there is:

```
LINK_CMD = $(QUIET_LINK) $(MKTGTDIR) ; $(CC) $(LDFLAGS) -o $@ $^ $(LIBS)
```